bundle/rhpam/manifests/rhpam-kogito-operator.clusterserviceversion.yaml

```diff
@@ -997,5 +997,5 @@ spec:
   maturity: beta
   provider:
     name: Red Hat
-  replaces: rhpam-kogito-operator.7.12.0-2
+  replaces: rhpam-kogito-operator.7.12.0-4
```

Comment:
you should now update only the version/rhpam/version.go file and run the bundle generation again
https://github.com/kiegroup/kogito-operator/blob/1.13.x/version/rhpam/version.go#L21.